src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -162,15 +176,68 @@ <h5><span jhiTranslate="artemisApp.studentExamDetail.exercises">Exercises</span>
                                 *ngIf="
                                     exercise?.studentParticipations &&
                                         exercise?.studentParticipations.length > 0 &&
-                                        (exercise?.studentParticipations)![0].results &&
-                                        (exercise?.studentParticipations)![0].results[0].assessor;
+                                        (exercise?.studentParticipations)![0].submissions &&
+                                        (exercise?.studentParticipations)![0].submissions[0].results &&
+                                        (exercise?.studentParticipations)![0].submissions[0].latestResult.assessor;
                                     else noAssessor
                                 "
                             >
-                                {{ (exercise?.studentParticipations)![0].results[0].assessor.name }}
+                                {{ (exercise?.studentParticipations)![0].submissions[0].latestResult.assessor.name }}
                             </ng-container>
                             <ng-template #noAssessor></ng-template>
                         </td>
+                        <td class="align-middle">
+                            <span *ngIf="exercise?.studentParticipations?.submissions?.length > 0 && exercise?.studentParticipations?.submissions[0].latestResult.hasComplaint"
+                                >Yes</span
+                            >
+                            <span *ngIf="!(exercise?.studentParticipations?.submissions?.length > 0 && exercise?.studentParticipations?.submissions[0].latestResult.hasComplaint)"
+                                >No</span
+                            >
+                        </td>
+                        <td>
+                            <div
+                                *ngIf="
+                                    course?.isAtLeastInstructor &&
+                                    exercise?.studentParticipations &&
+                                    exercise.type != ExerciseType.QUIZ &&
+                                    exercise.type != ExerciseType.PROGRAMMING
+                                "
+                            >
+                                <div *ngIf="(exercise?.studentParticipations)![0].results?.length > 0">
```

Comment:
you don't need to use brackets, in all cases: 

```suggestion
                                <div *ngIf="exercise?.studentParticipations![0].results?.length > 0">
```